Resources/Prototypes/Recipes/Lathes/Packs/medical.yml

```
@@ -107,3 +107,6 @@
   - StasisBedMachineCircuitboard
   - CryoPodMachineCircuitboard
   - BiomassReclaimerMachineCircuitboard
+  - CloningConsoleComputerCircuitboard # Carpmosia-start - civilian cloning research
+  - CloningPodMachineCircuitboard
+  - MedicalScannerMachineCircuitboard # Carpmosia-end - civilian cloning research
```

Start and end comments should surround instead of being inline. 
```suggestion
  # Carpmosia-start - civilian cloning research
  - CloningConsoleComputerCircuitboard
  - CloningPodMachineCircuitboard
  - MedicalScannerMachineCircuitboard
  # Carpmosia-end - civilian cloning research
```